.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,32 @@
+name: Report a bug
+description: Something is not working as expected? Tell us the problem
+labels: ["kind/bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for using Pelorus and reporting us back this problem, this helps
+        the project always improve!
+  - type: dropdown
+    attributes:
+      label: OpenShift version
+      description: In what version of OpenShift the problem ocurred?
+      options:
+        - 4.10
+        - 4.11
+        - 4.12
+        - 4.13
+        - Not related to OpenShift
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Problem description
+      description: |
+        Tell us what happened. The more info you provide, the faster the
+        problem can be fixed
+      placeholder: |
+        Fell free to add images and other files that can help us understand the
+        problem
+    validations:
+      required: true
```

Comment:
Adding another checkbox that points to our Code of Conduct is similar to the example at:

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

Reply:
done